web/themes/joinup/joinup_theme.theme

```diff
@@ -1400,6 +1406,12 @@ function joinup_theme_preprocess_rdf_entity(&$variables) {
   if ($variables['view_mode'] == 'about') {
     $variables['is_label_hidden'] = TRUE;
   }
+
+  // Add bundle class in listing templates.
```

Comment:
There is a clear repetition for solving the same task. Could be extracted into a function with a parameter being a reference variable to the `$variables`.

Reply:
Agree that this might be done better, but maybe let's create follow up ticket to review the main template, remove repetitions and create generic, reusable functions.